modules/ossm-updating-smcp.adoc

```diff
@@ -10,7 +10,7 @@ You can create or edit the `ServiceMeshControlPlane` with the command line.
 
 .Procedure
 
-. Log in to the {product-title} CLI as a user with the `cluster-admin` role. Enter the following command. Enter your username and password when prompted.
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role. If you are using {product-dedicated}, log in as a user with the `dedicated-admin` role. Enter the following command. Then, enter your username and password when prompted.
```

Comment:
I'm grimacing a little at the "Enter..." and "Then, enter..."

Maybe: "After you run the following command, enter your username and password when prompted:"  This *must* appear elsewhere in the doc set, but I can't spot it.

End the instructional text with a colon too?